Library/Formula/fdroidserver.rb

```diff
@@ -0,0 +1,104 @@
+  end
+
+  resource "backports.ssl_match_hostname" do
+    url "https://pypi.python.org/packages/source/b/backports.ssl_match_hostname/backports.ssl_match_hostname-3.4.0.2.tar.gz"
+    sha256 "07410e7fb09aab7bdaf5e618de66c3dac84e2e3d628352814dc4c37de321d6ae"
+  end
+
+  resource "mwclient" do
+    url "https://pypi.python.org/packages/source/m/mwclient/mwclient-0.7.2.zip"
+    sha256 "195462115fadbe3cee73b5a0fe3cc28c8bdc0115c590cdc0f8b5083861003d1a"
+  end
+
+  resource "paramiko" do
+    url "https://pypi.python.org/packages/source/p/paramiko/paramiko-1.15.2.tar.gz"
+    sha256 "4f56a671a3eecbb76e6143e6e4ca007d503a39aa79aa9e14ade667fa53fd6e55"
+  end
+
+  resource "pillow" do
+    url "https://pypi.python.org/packages/source/P/Pillow/Pillow-2.9.0.tar.gz"
+    sha256 "0f179d7e75e7c83b6341b9595ca1f394de7081484a9e352ad66d553a1c3daa29"
+  end
+
+  resource "pyasn1" do
+    url "https://pypi.python.org/packages/source/p/pyasn1/pyasn1-0.1.7.tar.gz"
+    sha256 "e4f81d53c533f6bd9526b047f047f7b101c24ab17339c1a7ad8f98b25c101eab"
+  end
+
+  resource "pyasn1-modules" do
+    url "https://pypi.python.org/packages/source/p/pyasn1-modules/pyasn1-modules-0.0.5.tar.gz"
+    sha256 "be65f00ed28e30756f1ef39377cb382480a2368699179d646a84d79fe9349941"
+  end
+
+  resource "python-magic" do
+    url "https://pypi.python.org/packages/source/p/python-magic/python-magic-0.4.6.tar.gz"
+    sha256 "903d3d3c676e2b1244892954e2bbbe27871a633385a9bfe81f1a81a7032df2fe"
+  end
+
+  resource "PyYAML" do
+    url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
+    sha256 "c36c938a872e5ff494938b33b14aaa156cb439ec67548fcab3535bb78b0846e8"
+  end
+
+  resource "requests" do
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.7.0.tar.gz"
+    sha256 "398a3db6d61899d25fd4a06c6ca12051b0ce171d705decd7ed5511517b4bb93d"
+  end
+
+  def install
+    ENV["PYTHONPATH"] = Formula["libdnet"].opt_lib/"python2.7/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    ENV.append_to_cflags "-I#{Formula["freetype"].opt_include}/freetype2"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  def caveats; <<-EOS.undent
+    In order to function, fdroidserver requires that the Android SDK's
+    "Build-tools" and "Platform-tools" are installed.  Also, it is best if the
+    base path of the Android SDK is set in the standard environment variable
+    ANDROID_HOME.  To install them from the command line, run:
+    android update sdk --no-ui --filter platform-tools,build-tools-22.0.1
+    EOS
```

Comment:
I looked into doing this.  There are a lot of notes and caveats so its a complicated thing.  I don't currently have time to take on that project.  For example:

    # The 'android' tool insists on deleting #{prefix}/platform-tools
    # and then installing the new one. So it is impossible for us to redirect
    # the SDK location to var so that the platform-tools don't have to be
    # freshly installed EVERY DANG time the base SDK updates.